src/components/gameUtils.js

```diff
@@ -28,23 +34,27 @@ const letterDistribution = {
 	Z: 1,
 };
 export const createInitialLetterBag = () => {
-	let initialLetterBag = [];
-	for (const [letter, count] of Object.entries(letterDistribution)) {
-		for (let i = 0; i < count; i++) {
-			initialLetterBag.push(letter);
-		}
-	}
-	return initialLetterBag;
+	// let initialLetterBag = [];
+	// for (const [letter, count] of Object.entries(letterDistribution)) {
+	// 	initialLetterBag.push([letter, count]);
+	// }
```

Comment:
I refactored the code to rely on the dictionary above, instead of an array of arrays. You can get that any time by re-using `Object.entries(letterDistribution)` but the dictionary is handy for reducing the count.